mobile-ui/src/main/java/com/playone/mobile/ui/ext/Activity.kt

```diff
@@ -0,0 +1,23 @@
+package com.playone.mobile.ui.ext
+
+import android.content.Context
+import android.content.Intent
+import android.support.v4.app.FragmentTransaction
+import android.support.v7.app.AppCompatActivity
+
+
+inline fun <reified T : Any> AppCompatActivity.startForResult(
+        requestCode: Int = -1,
+        noinline init: Intent.() -> Unit = {}) {
+
+    val intent = newIntent<T>(this)
+    intent.init()
+    startActivityForResult(intent, requestCode)
+}
+
+inline fun <reified T : Any> newIntent(context: Context): Intent =
```

Comment:
I always omit the return data type. I don't know it's good or not 🧐

Reply:
fixed. Here is not needed.